installer/ServerInstaller.cs

```diff
@@ -88,7 +88,7 @@ class BuildInstaller
 		bool isAllLangs = false;
 
 		for (int i = 0; i < args.Length; i++) {
-			if (args[i] == "--version" && Regex.IsMatch(args[i + 1], @"^[0-9]\.[0-9]\.[0-9]\.[0-9]$")) {
+			if (args[i] == "--version" && Regex.IsMatch(args[i + 1], @"^([0-9]+\.){3}[0-9]+$")) {
```

Comment:
The RegEx is duplicated here. We should instead outsource this check into a function so that we only have to maintain a single version of this RegEx